src/main/scala/org/mbari/vars/annotation/controllers/ImageController.scala

```diff
@@ -83,6 +84,7 @@ class ImageController(daoFactory: BasicDAOFactory) {
       val imagedMoment = ImagedMomentController
         .findOrCreateImagedMoment(imDao, videoReferenceUUID, timecode, recordedDate, elapsedTime)
       val imageReference = irDao.newPersistentObject(url, description, height, width, format)
+      imageReferenceUUID.foreach(uuid => imageReference.uuid = uuid)
```

Comment:
@kevinsbarnard Options (and other monad-y/container-y things in Scala) all have methods like `foreach`, `map`, and `flatmap`, etc to help do something with the object. The an also be pattern matched on, for example this does the same as `imageReferenceUUID.foreach(uuid => imageReference.uuid = uuid)`

```scala
imageReferenceUUID match {
  case Some(uuid) => imageReference.uuid = uuid
  case None => // Do nothing
}
```